tests/ascii_data_adiabatic_conditions/depth_average.gnuplot

```diff
@@ -4,74 +4,74 @@
 #
 # For a description of the GNUPLOT format see the GNUPLOT manual.
 #
-# <depth> <time> <temperature> <adiabatic_temperature> <adiabatic_pressure> <adiabatic_density> <adiabatic_density_derivative> <velocity_magnitude> <sinking_velocity> <viscosity> <vertical_heat_flux> 
-0 0 3.05 3 1 5 3.46021e-07 3.17701e-13 3.0917e-14 1e+21 1.6263e-06 
-289000 0 3.05 3 1 5 3.46021e-07 3.17701e-13 3.0917e-14 1e+21 1.6263e-06 
+# <depth> <time> <temperature> <adiabatic_temperature> <adiabatic_pressure> <adiabatic_density> <adiabatic_density_derivative> <velocity_magnitude> <sinking_velocity> <viscosity> <vertical_heat_flux> <vertical_mass_flux> 
+0 0 3.05 3 1 5 3.46021e-07 3.17104e-13 3.08398e-14 1e+21 1.6263e-06 6.89975e-18 
```

Comment:
Any idea why these numbers have changed? That must have happened in an earlier change, and is probably so small that numdiff is fine with it?

Reply:
Yes that is the best explanation I have as well.